tests/test_baseclass.py

```diff
@@ -178,25 +179,26 @@ def test_model_selection_with_pipeline_works_as_expected(
     def test_regression_model_can_be_saved(
         self, classifier, tmp_path, base, test_dataset
     ):
-        expected_path = tmp_path / "test_model.pkl"
-
         classifier.score_estimator(test_dataset)
-        classifier.save_estimator(expected_path)
+        load_storage = FileStorage(tmp_path)
+
+        with FileStorage(tmp_path) as storage:
+            saved_model_path = classifier.save_estimator(storage)
 
-        assert expected_path.exists()
+            assert saved_model_path.exists()
 
-        loaded_model = base.load_estimator(str(expected_path))
-        assert loaded_model.estimator.get_params() == classifier.estimator.get_params()
+            loaded_model = base.load_estimator(load_storage, saved_model_path)
+            assert loaded_model.estimator.get_params() == classifier.estimator.get_params()
```

Comment:
Do we have a test where we check that we can make a prediction with the loaded estimator?

Reply:
no :) we have them separate though, loading and prediction